src/content/1.7/development/architecture/domain/value_objects.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Value Objects
+weight: 20
+---
+
+# Value Objects
+
+PrestaShop is using Value Objects in its codebase, to help you understand better what they are, this article aims to explain what Value object is and how to use it.
+
+## What is Value Object?
+
+A Value Object is small immutable object whos equality is not based on identity, this means that two Value Objects are equal when they have the same value, but not necessarily being the same object.
+
+Characteristics of Value Object are:
+
+* It does not have identity (e.g. `Money` is value object because it can be created without id like `new Money($amount)`).
```

Comment:
```suggestion
* It does not have an identity, contrary to an Entity (e.g. `Money($amount)` can be considered a VO because two instances of it can be considered equal as long as the `$amount` is equal, regardless of which bills that amount is made of; but `Bill($billNumber)` is an entity because it refers to a specific physical Bill).
```